net-tools/netwatch/src/netmon/actor.rs

```diff
@@ -102,7 +101,7 @@ impl Actor {
         self.actor_sender.clone()
     }
 
-    pub(super) async fn run(mut self) {
+    pub(super) async fn run(mut self) -> Result<()> {
```

Comment:
I'm confused, the signature is changed but there's not a single error return?

Reply:
yeah, let's revert as then the supervisor discussion goes away as well :)

Reply:
done